### PR TITLE
may the Mirror overlords finally have mercy on this PR

### PR DESCRIPTION
**Pull request by MaxIsJoe**

### CODE REVIEW

File: UnityProject/Assets/Scripts/Core/Lighting/LightsHolder.cs [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+			}
+		}
+	}
+
+	public struct LightData
+	{
+		public int Id;
+		public Color lightColor;
+		public Light2D.LightSprite.LightShape lightShape;
+		public GameObject lightSpriteObject;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Core/Lighting/LightsHolder.cs]: by the way it will try networking this

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Core/Lighting/LightsHolder.cs]: wdym?
I am trying to network it, whats the issue?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Core/Lighting/LightsHolder.cs]: you can't network game objects

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Core/Lighting/LightsHolder.cs]: https://mirror-networking.gitbook.io/docs/manual/guides/data-types

**Comment by MaxIsJoe** [comment]
why does tests keep failing

**Comment by Bod9001** [comment]
compile errors?

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**